js/core/component.js

```diff
@@ -51,7 +51,7 @@ var Component = Class.inherit({
             * @name ComponentOptions.onInitialized
             * @type function
             * @type_function_param1 e:object
-            * @type_function_param1_field1 component:Component
+            * @type_function_param1_field1 component:this
```

Comment:
The 'this' string as a type looks strange for me on the documentation comments side.